core/profile.cpp

```diff
@@ -1368,8 +1368,8 @@ static std::vector<std::string> plot_string(const struct dive *d, const struct p
 	std::vector<std::string> res;
 
 	depthvalue = get_depth_units(entry->depth, NULL, &depth_unit);
-	res.push_back(casprintf_loc(translate("gettextFromC", "@: %d:%02d"), FRACTION(entry->sec, 60), depthvalue));
-	res.push_back(casprintf_loc(translate("gettextFromC", "D: %.1f%s"), depth_unit));
+	res.push_back(casprintf_loc(translate("gettextFromC", "@: %d:%02d"), FRACTION(entry->sec, 60)));
```

Comment:
I can see how a macro (having function syntax) returning a tuple is basically inviting this kind of bug - might be helpful to make it a bit more explicit by calling it `FRACTION_TUPLE()` or similar.

Reply:
Well, I tripped over that macro a few times already, so I should have known better. 🤷